simpleflow/swf/task.py

```diff
@@ -220,6 +217,13 @@ def schedule(self, domain, task_list=None, executor=None, **kwargs):
 
         return [decision]
 
+    def get_input(self):
+        input = {
+            'args': self.args,
+            'kwargs': self.kwargs,
```

Comment:
Always pass the "meta" key too. Unsure if it makes sense here but I think we'll use it more and more for plumbing.

Reply:
Where's `meta`? I remember you wanted to have it along args/kwargs but IIRC it's in currently in kwargs?

Reply:
You're right, "meta" is at the same level as "args" / "kwargs" for activities, but for workflows it's not used yet. It could be though, for instance for passing extra control information without having to change workflow signatures. We do have the habit of supporting "**kwags" at Botify but it shouldn't be mandatory.

Example usage: supporting more than 5 "tags" via a "meta.additional_tags = [t1, t2, ...]" ; those tags wouldn't be usable in SWF search endpoints, but we don't care much, they could be useful for passing extra info anyway that shouldn't bother the actual work of the workflow.

Reply:
Counter-point: it's not implemented anywhere now, and in my review I mixed activities and workflows. Maybe don't change anything until we implement what I say?

Reply:
Right, let's do it when we do it :slightly_smiling_face: